### PR TITLE
Migrate to express + postgres

### PR DESCRIPTION
**Pull request by bwendt-mylo**

### CODE REVIEW

**Comment by bwendt-mylo** [comment]
@bmarks-mylo I need help making the `develop` branch the default branch for this repo, as well as getting this hooked up to travis. I've added the travis.yml file but can't seem to get this repo to show up on travis-ci.com.

File: README.md [hunk 1]
```diff
@@ -1,17 +1,36 @@
 # mylo-rcr
 
-Running via Firebase cloud functions.
+## Purpose
```
Comment (angela-mylo) [inline README.md]: Love seeing purpose & accountabilities!  :fireworks:

File: app/app.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+app.use('/healthz', diag.healthy(healthCheck))
+app.use(mylog.logHttp)
+
+app.post('/slack', checkSlackTokenMiddleware, slackRoute)
+
+app.use(mylog.logHttpError)
+
+module.exports = app
+
+// NOTE - This only works for postgres databases
```
Comment (angela-mylo) [inline app/app.js]: Probably can remove these comments

**Comment by angela-mylo** [comment]
@bwendt-mylo Someone beat me to the branch settings; looks like develop is default now.

**Comment by bmarks-mylo** [comment]
going to close and open a new PR to try and get Travis to pick it up

**Pull request closed without merging**